docs/index.rst

```diff
@@ -56,36 +56,28 @@ Tests with twine are run using `tox`_, and tested against the following Python
 versions: 2.7, 3.4, 3,5, and 3.6. To run these tests locally, you will need to
 have these versions of Python installed on your machine.
 
-If you are using pipenv to manage your virtual environment, you may need the
-`tox-pipenv`_ plugin so that tox can use pipenv environments instead of
-virtualenvs.
+If you are using ``pipenv`` to manage your virtual environment, you
+may need the `tox-pipenv`_ plugin so that tox can use pipenv
+environments instead of virtualenvs.
 
 Building the documentation
 --------------------------
 
 Additions and edits to twine's documentation are welcome and appreciated. To
 build the docs locally, first set up a virtual environment and activate it,
-using Python 3.6 as the Python version in the virtual environment. Then install
-tox:
+using Python 3.6 as the Python version in the virtual environment. Example:
 
 .. code-block:: console
 
-  pip install tox
+  mkvirtualenv -p /usr/bin/python3.6 twine
 
-Install Sphinx:
+Then install the required dependencies:
```

Comment:
I missed this in the initial review. You don't need to install those to run `tox -e docs`.

Instead, let's remove this and rewrite this to just be

```
Then, build the docs using ``tox``:
```

And we can update `tox.ini` to install from our requirements file so everything works the same way.

Reply:
Thanks for the suggestion, @sigmavirus24. I looked at [the tox documentation](https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints) and believe I've now successfully done this. f298a8ee8e271994020b1d6d9dbade6b62e24a2d builds fine on my machine in a virtualenv that does not have Sphinx, *et alia* installed, and the fresh build of [my test project on RtD](https://my-fork-of-twine.readthedocs.io/en/fix-rtd/) built successfully. So, ready for re-review.